### PR TITLE
OSDOCS-5318: remove fixed iptable and node port workarounds

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5318
also https://issues.redhat.com/browse/OCPBUGS-6908

Link to docs preview:
https://56610--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshooting.html 

QE review:
- [x] QE has approved this change.

Additional information:
https://access.redhat.com/errata/RHSA-2023:0772 advisory verifying fix

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@jogeo Did QE verify this fix?

**Comment by ShaunaDiaz** [comment]
@zshi-redhat PTAL and let me know if correct to remove both workarounds.

File: microshift_troubleshooting/microshift-troubleshooting.adoc [hunk 1]
```diff
@@ -8,8 +8,10 @@ toc::[]
 
 Read about troubleshooting and possible solutions for known issues.
 
-include::modules/microshift-ki-cni-iptables-deleted.adoc[leveloffset=+1]
+//include::modules/microshift-ki-cni-iptables-deleted.adoc[leveloffset=+1]
 
-include::modules/microshift-troubleshooting-nodeport.adoc[leveloffset=+1]
+//include::modules/microshift-troubleshooting-nodeport.adoc[leveloffset=+1]
```
Comment (zshi-redhat) [inline microshift_troubleshooting/microshift-troubleshooting.adoc:13]: This (troubleshooting-nodeport) should not be removed.

File: microshift_troubleshooting/microshift-troubleshooting.adoc [hunk 2]
```diff
@@ -8,8 +8,10 @@ toc::[]
...
 
 include::modules/microshift-nodeport-unreachable-workaround.adoc[leveloffset=+1]
```
Comment (zshi-redhat) [inline microshift_troubleshooting/microshift-troubleshooting.adoc:15]: This line (nodeport-unreachable-workaround) can be commented out.

**Pull request merged**